crates/autopilot/src/limit_order_quoter.rs

```diff
@@ -57,63 +58,70 @@ impl LimitOrderQuoter {
             if orders.is_empty() {
                 break;
             }
-            for order in orders {
-                match self
-                    .quoter
-                    .calculate_quote(QuoteParameters {
-                        sell_token: order.data.sell_token,
-                        buy_token: order.data.buy_token,
-                        side: match order.data.kind {
-                            OrderKind::Buy => OrderQuoteSide::Buy {
-                                buy_amount_after_fee: order.data.buy_amount,
-                            },
-                            OrderKind::Sell => OrderQuoteSide::Sell {
-                                sell_amount: SellAmount::BeforeFee {
-                                    value: order.data.sell_amount + order.data.fee_amount,
+            for chunk in orders.chunks(10) {
+                ignored_orders += stream::iter(chunk)
+                    .map(|order| async move {
```

Comment:
Does `stream::iter` actually start polling each future in concurrently, or does it wait for the previous item to resolve before polling the next one (what I naively would think it does)?

Reply:
I think the way to go here is to turn each item of the chunk into a future and throw those futures into `futures::future::join_all(futures)` to execute them in parallel.